gladier/client.py

```diff
@@ -602,15 +605,20 @@ def run_flow(self, flow_input=None, use_defaults=True):
                     log.info('Initiating new login for dependent scope change')
                     self.login(requested_scopes=[cfg_sec['flow_scope']], force=True)
                     flow = self.flows_client.run_flow(flow_id, cfg_sec['flow_scope'],
-                                                      combine_flow_input, **flow_permissions).data
+                                                      combine_flow_input, label=flow_label,
+                                                       **flow_permissions).data
                 else:
                     raise gladier.exc.AuthException('Scope change for flow, re-auth required',
                                                     missing_scopes=(cfg_sec['flow_scope'],))
             else:
                 raise
-        log.info(f'Started flow {self.section} flow id "{cfg_sec["flow_id"]}" with action '
-
+        if flow_label:
+            log.info(f'Started flow {flow_label} flow id "{cfg_sec["flow_id"]}" with action '
+                 f'"{flow["action_id"]}"')
+        else:
+            log.info(f'Started flow {self.section} flow id "{cfg_sec["flow_id"]}" with action '
```

Comment:
If the above is changed to `flow_kwargs`, could we change this to pass the following instead:
```
log.info(f'Started flow {flow_kwargs.get("label")} flow id "{cfg_sec["flow_id"]}" with action '
                 f'"{flow["action_id"]}"')
```
And remove the extra `if` statement?